exercises/minesweeper/tests/minesweeper.rs

```diff
@@ -22,121 +26,71 @@ fn run_test(test_case: &[&str]) {
 
 #[test]
 fn empty_board_has_no_annotations() {
-    run_test(&[
-        "   ",
-        "   ",
-        "   "
-    ]);
+    run_test(&["   ", "   ", "   "]);
```

Comment:
these make it a bit harder to visualize the boards, but I hate to have to fight rustfmt. Any way to tell rustfmt "no, really, keep these as they are"?